### PR TITLE
Adding `rexmex` from PyPI

### PR DESCRIPTION
**Pull request by sugatoray**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by sugatoray** [comment]
## Notes

Since `pip-check` fails as it looks for `sklearn` (despite installing scikit-learn), pip-check was dropped.

**Comment by sugatoray** [comment]
PR is ready. Request for review/merging.

cc: @conda-forge/help-python

File: recipes/rexmex/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    - scikit-learn
+
+test:
+  imports:
+    - rexmex
+    - rexmex.metrics
+  # commands:
+  #   - pip check
+  requires:
+    - pip
```
Comment (marcelotrevisani) [inline recipes/rexmex/meta.yaml:38]: as ``pip`` cannot be used here can you delete these lines please?
```suggestion
```

File: recipes/rexmex/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+    - pytest-runner
+    - python >=3.6
+  run:
+    - python >=3.6
+    - numpy
+    - pandas <=1.3.5
+    - scikit-learn
+    - scipy
+    - scikit-learn
```
Comment (marcelotrevisani) [inline recipes/rexmex/meta.yaml:29]: duplicated package
```suggestion
```

**Review by marcelotrevisani — APPROVED** [review]

**Comment by sugatoray** [comment]
@marcelotrevisani Done. Thank you.

**Comment by marcelotrevisani** [comment]
I am just waiting for azure to be green and I will merge it, thanks for your contribution! :)

**Pull request merged**